src/main/java/com/meetup/controller/TalkController.java

```diff
@@ -0,0 +1,48 @@
+package com.meetup.controller;
+
+import com.meetup.service.TalkService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/talk")
+public class TalkController {
+  @Autowired
+  private TalkService service;
+
+  //@PostMapping(produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Will get rid of this talk service eventually. just left it here as a placeholder if would want to separate it out to it's own endpoint